tests/beaker_tests/cisco_vrf_af/test_vrf_af.rb

```diff
@@ -37,22 +37,28 @@
   manifest_props: {
     route_target_both_auto:      'default',
     route_target_both_auto_evpn: 'default',
+    route_target_both_auto_mvpn: 'default',
     route_target_import:         'default',
     route_target_import_evpn:    'default',
+    route_target_import_mvpn:    'default',
     route_target_export:         'default',
     route_target_export_evpn:    'default',
+    route_target_export_mvpn:    'default',
   },
   resource:       {
     'route_target_both_auto'      => 'false',
     'route_target_both_auto_evpn' => 'false',
+    'route_target_both_auto_mvpn' => 'false',
   },
 }
 
 routetargetimport          = ['1.1.1.1:55', '1:33']
 routetargetimportevpn      = ['2.2.2.2:55', '2:33']
+routetargetimportmvpn      = ['7.7.7.7:55', '7:33']
```

Comment:
We are sorting this list in both the type and node_utils right?  Just making sure we will be idempotent regardless of what order these are specified in.

Reply:
Yes, this works even with the list reversed and idempotence is maintained.